ignite/services/network/publish.go

```diff
@@ -14,16 +18,54 @@ import (
 	"github.com/ignite-hq/cli/ignite/services/network/networktypes"
 )
 
+// SharePercentage represent percent of total share
+type SharePercentage struct {
```

Comment:
I am not sure but I believe that semantically would be also fine to use "percent" where single values are involved, so `SharePercentage` could be `SharePercent`. Plurals could still be defined as "percentages".